### PR TITLE
[NUI] fixed bug about set desired size 

### PR DESCRIPTION
**Pull request by JoogabYun**
### Description of Change ###
[NUI] fixed bug about set desired size type to FittingModeType.Fill

### API Changes ###
none

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/ImageView.cs [hunk 1]
```diff
@@ -1145,10 +1140,15 @@ private void UpdateImage(int key, PropertyValue value)
                   }
                   temp.Insert(ImageVisualProperty.DesiredWidth, new PropertyValue((int)ret_width));
                   temp.Insert(ImageVisualProperty.DesiredHeight, new PropertyValue((int)ret_height));
-                  temp.Insert(ImageVisualProperty.FittingMode, new PropertyValue((int) FittingModeType.ShrinkToFit));
+                  temp.Insert(ImageVisualProperty.FittingMode, new PropertyValue((int) FittingModeType.Fill));
```
Comment (bshsqa) [inline src/Tizen.NUI/src/public/BaseComponents/ImageView.cs:1143]: Size2D imageSize = ImageLoading.GetOriginalImageSize(_resourceUrl);

                  int adjustedDesiredWidth, adjustedDesiredHeight;
                  float aspectOfDesiredSize = (float)_desired_height / (float)_desired_width;
                  float aspectOfImageSize = (float)imageSize.Height / (float)imageSize.Width;
                  if( aspectOfImageSize > aspectOfDesiredSize)
                  {
                      adjustedDesiredWidth = _desired_width;
                      adjustedDesiredHeight = imageSize.Height * _desired_width /(imageSize.Width);
                  }
                  else
                  {
                      adjustedDesiredWidth = imageSize.Width * _desired_height/(imageSize.Height);
                      adjustedDesiredHeight = _desired_height;

                  }
                  temp.Insert(ImageVisualProperty.DesiredWidth, new PropertyValue((int)adjustedDesiredWidth));
                  temp.Insert(ImageVisualProperty.DesiredHeight, new PropertyValue((int)adjustedDesiredHeight));
                  temp.Insert(ImageVisualProperty.FittingMode, new PropertyValue((int) FittingModeType.ShrinkToFit));

File: src/Tizen.NUI/src/public/BaseComponents/ImageView.cs [hunk 2]
```diff
@@ -1131,20 +1131,21 @@ private void UpdateImage(int key, PropertyValue value)
...
                   {
-                      ret_width = imageSize.Width*_desired_width/(imageSize.Height);
-                      ret_height = _desired_height;
-
+                      adjustedDesiredWidth = imageSize.Width * _desired_height/(imageSize.Height);
+                      adjustedDesiredHeight = _desired_height;
                   }
-                  temp.Insert(ImageVisualProperty.DesiredWidth, new PropertyValue((int)ret_width));
-                  temp.Insert(ImageVisualProperty.DesiredHeight, new PropertyValue((int)ret_height));
+                  temp.Insert(ImageVisualProperty.DesiredWidth, new PropertyValue((int)adjustedDesiredWidth));
```
Comment (bshsqa) [inline src/Tizen.NUI/src/public/BaseComponents/ImageView.cs:1147]: Sorry. this is already int type.

File: src/Tizen.NUI/src/public/BaseComponents/ImageView.cs [hunk 3]
```diff
@@ -1131,20 +1131,21 @@ private void UpdateImage(int key, PropertyValue value)
...
-                  if( imageSize.Width > imageSize.Height)
+                  int adjustedDesiredWidth, adjustedDesiredHeight;
+                  float aspectOfDesiredSize = (float)_desired_height / (float)_desired_width;
+                  float aspectOfImageSize = (float)imageSize.Height / (float)imageSize.Width;
+                  if( aspectOfImageSize > aspectOfDesiredSize)
                   {
-                      ret_width = _desired_width;
-                      ret_height = imageSize.Height * _desired_height /(imageSize.Width);
+                      adjustedDesiredWidth = _desired_width;
+                      adjustedDesiredHeight = imageSize.Height * _desired_width /(imageSize.Width);
```
Comment (bshsqa) [inline src/Tizen.NUI/src/public/BaseComponents/ImageView.cs:1140]: Sorry, we don't need to use bracket.

File: src/Tizen.NUI/src/public/BaseComponents/ImageView.cs [hunk 4]
```diff
@@ -881,8 +881,8 @@ public int DesiredWidth
             }
             set
             {
-                UpdateImage(ImageVisualProperty.DesiredWidth, new PropertyValue(value));
                 _desired_width = value;
```
Comment (bshsqa) [inline src/Tizen.NUI/src/public/BaseComponents/ImageView.cs:884]: It maybe better.
if (_desired_width != value)
{
    _desired_width = value;
    UpdateImage(0, null);
}

This is same for the DesiredHeight.

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Comment by dongsug-song** [comment]
@bshsqa 
Hello, please review again.
thank you.

**Review by bshsqa — APPROVED** [review]
Looks good to me.

**Pull request merged**